samtranslator/model/eventsources/push.py

```diff
@@ -743,6 +743,24 @@ def to_cloudformation(self, **kwargs):  # type: ignore[no-untyped-def]
         if explicit_api.get("__MANAGE_SWAGGER") or explicit_api.get("MergeDefinitions"):
             self._add_swagger_integration(explicit_api, api_id, function, intrinsics_resolver)  # type: ignore[no-untyped-call]
 
+        swagger_body = explicit_api.get("DefinitionBody")
+
+        if swagger_body and self.Auth and self.Auth.get("OverrideApiAuth") is True:
```

Comment:
[nit] Is the `is True` necessary?

Reply:
Good catch, I don't think it is. I'll push a change to remove this.